src/components/overview/ImageGallery/DefaultView.jsx

```diff
@@ -1,43 +1,26 @@
 import React, { useState, useEffect } from 'react';
 
-const DefaultView = ({state, updateState}) => {
-  const [ mainImage, setMainImage ] = useState('');
-
-  const updateMainImage = (event) => {
-    event.preventDefault();
-    let clickedImg = event.target.src;
-    setMainImage(clickedImg);
-  };
-
-  const handleToExpand = () => {
-    let mainImageElement = document.getElementById('main');
-    mainImageElement.style.width = '75rem';
-  };
+const DefaultView = ({state, updateState, updateMainImage, handleToExpand, handleRightArrow, handleLeftArrow}) => {
 
   return (
-    <div className='defaultView'>
-      <div className='styleImages'>
-        {state.styleImages.map((element, idx) => { //render all style photos
-          return <img
-            style = {{
-              width: '9vh',
-              height: '9vh',
-              cursor: 'pointer'
-            }}
-            id = {idx}
-            onClick = {updateMainImage}
-            src={element.url}/>;
-        })}
-      </div>
-      <div className='mainImage'>
-        <img className='btnFloating' id='left' src ="https://img.icons8.com/ios-filled/50/000000/long-arrow-left.png"></img>
-        {(state.styleImages[0] !== undefined && mainImage === '') ?
-          <img id = 'main' onClick = {handleToExpand} src = {state.styleImages[0].url} style = {{width: '75vh', height: '54vh'}}/> :
-          <img id = 'main' onClick = {handleToExpand} src = {mainImage} style = {{width: '75vh', height: '54vh'}}/>
-        }
-        <img className='btnFloating' id='right' src ="https://img.icons8.com/ios-filled/50/000000/long-arrow-right.png"></img>
-      </div>
-    </div>
+    <>
+      {(state !== undefined) && (
+        <div className='defaultView'>
+          <div className='styleImages'>
+            {state.styleImages.map((element, idx) => { //render all style photos
+              return <img id={idx} key={idx} className='eachStyleImage' onClick={updateMainImage} src={element.url}/>; })}
```

Comment:
Idea here- for elements with more than a couple props, perhaps break the props into a multiple lines? It looks a little cleaner/easier to read and it seems that the other modules have adopted this styling, so I think it would be great to help keep our code looking uniform

Reply:
I agree with this - I've been doing it in all my code, and I think it's just good coding practice.

Reply:
Thanks for letting me know! I actually broke the props into a multiple lines before and then merged into one line because I thought it's getting too long.. But breaking into a multiple lines definitely seems easier to read as you said. 
It is changed now!